src/Domain/CoordinateMaps/ProductMaps.hpp

```diff
@@ -117,9 +117,8 @@ class ProductOf2Maps {
   void pup(PUP::er& p);  // NOLINT
 
  private:
-  template <typename Map1Op, typename Map2Op>
-  friend bool operator==(const ProductOf2Maps<Map1Op, Map2Op>& lhs,
-                         const ProductOf2Maps<Map1Op, Map2Op>& rhs) noexcept {
+  friend bool operator==(const ProductOf2Maps<Map1, Map2>& lhs,
```

Comment:
General question for our knowledge: Do you need to specify the template parameters here?

Reply:
If we don't, should they be removed?

Reply:
And should I also update the inequality operators in that case?

Reply:
yes and yes

Reply:
It seems like the inequality operators still need the template parameters, but for the equality operators they can be removed

Reply:
ah right, they are not in the class, so they will still need the template parameters